source/includes/_protocol_schemas.mdown

```diff
@@ -0,0 +1,217 @@
+# Protocol Schemas
+
+
+## Listing Schema
+
+This describes a listing for purchase on the Origin marketplace.
+
+```javascript
+{
+    schemaId: "http://schema.originprotocol.com/listing_v1.0.0",
+    listingType: "unit",
+    category: "schema.forSale",
+    subCategory: "schema.forSale.carsTrucks",
+    language: "en-us",
+    title: "1997 Lotus Esprit",
+    description: "Red. 100K miles. Always garaged.",
+    expiry: "2018-09-20T19:59:36.875Z",
+    media: [
+        {url: "ipfs://Ac92fa25B89D37077C7e52BDae22094f7b864e91"},
+        {url: "ipfs://C9a2fa25B89D37045C7e52BDae22094f7b864b82"},
+    ],
+    price: {amount: 48.7, currency: "ETH"},
+    unitsTotal: 1,
+    securityDeposit: {amount: 500, currency: "OGN"},
+    commission: {amount: 500, currency: "OGN"},
+}
+```
+
+### Required Fields:
+
+| | |
+|----|-----|
+|**schemaId** | Must be set to `http://schema.originprotocol.com/listing_v1.0.0` |
+|**listingType** | Must be set to `unit` |
+|**category** | A category/schema key string from the DApp. See possible values [here](https://github.com/OriginProtocol/origin-dapp/tree/master/src/schemaMessages)  |
+|**subCategory** | A subcategory key string from a schema in the DApp. See possible values inside each DApp schema [here](https://github.com/OriginProtocol/origin-dapp/tree/master/src/schemaMessages)
+|**language** | A single language or locale tag that the listing text is written in. |
+|**title** | The short title of the listing |
+|**description** | A full textual description of the listing. Linebreaks are currently ignored, though this will change. |
+|**price** | For currencies, we support `ETH`, with future plans to support any ERC20 token |
+|**unitsTotal** | The number of units available for sale. Currently limited to `1` |
+
+### Optional Fields
+
+| | |
+|----|-----|
+|expiry | A date that this listing should no longer show in search results. This is suggestion and search engines may ignore it.|
+|media | A list of media elements to be shown with the listing. We currently only support **images** on the **ipfs://** protocol.|
+|securityDeposit| **UNSAFE FIELD - this data duplicates, and may not match, the actual blockchain contract amount.** The currency must be `OGN`. |
```

Comment:
AFAICT we are not populating this field and it is not a copy of a value from the blockchain.
I added this a while back for future use (to store the amount the seller would ask as security deposit). But it was not really thought through. I'm thinking I should remove it from the V1 schema for now. Thoughts ?